include/net/net_pkt.h

```diff
@@ -1311,6 +1304,26 @@ int net_pkt_split(struct net_pkt *pkt, struct net_buf *orig_frag,
 		  struct net_buf **fragB, s32_t timeout);
 
 /**
+ * @brief Return the fragment and offset within it according to network
+ * packet offset.
+ *
+ * @details This is typically used to get the protocol header pointer that
+ * we know the offset. According to this information, the corresponding fragment
+ * and offset within that fragment is returned.
+ *
+ * @param pkt Network packet
+ * @param offset Offset of desired location in network packet.
+ * @param pos Pointer to position of offset after reading n number of bytes,
```

Comment:
> after reading n number

What's "n"? I guess this needs to be reworded to be more clear."Pointer to position within result fragment corresponding to offset param" or something.